incubator/kube-aws-ingress-controller/templates/daemonset.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: extensions/v1beta2
+kind: DaemonSet
+metadata:
+  name: {{ template "kube-aws-ingress-controller.fullname" . }}
+  namespace: kube-system
```

Comment:
Any reason the namespace is hard-coded to `kube-system`? In general, it is considered bad practice to install stuff into `kube-system`.

Reply:
I've taken this from the official manifest file coming on the project: https://github.com/zalando-incubator/kube-ingress-aws-controller/tree/master/deploy

I know it's bad, but if they "officially" require it, it's maybe because it's needed there. I would leave this to any Zalando member honestly.